content/docs/user-guide/packaging/project-export/project-export-cli.md

```diff
@@ -17,6 +17,24 @@ To make best use of the `export-project` command, it is recommended to have a pr
 To learn more about the AssetBundler and Seed Files, please visit the [overview page on the AssetBundler tool](https://docs.o3de.org/docs/user-guide/packaging/asset-bundler/overview/).
 
 ## Getting Started
+### TL;DR
+For any new project that you quickly want to test with export, assuming you are using the Default Level supplied with the project as your base, you can quickly do this by double clicking on the [`export.bat`](https://github.com/o3de/o3de/blob/480ce34ab7f3056113e17056125c89ae220bef7a/Templates/DefaultProject/Template/export.bat) script (or running the [`export.sh`](https://github.com/o3de/o3de/blob/480ce34ab7f3056113e17056125c89ae220bef7a/Templates/DefaultProject/Template/export.sh) script on Linux) in the root of your Project folder.
```

Comment:
```suggestion
For any project that you want to export, assuming you are using the Default Level supplied with the template as your base, you can do this by running the [`export.bat`](https://github.com/o3de/o3de/blob/480ce34ab7f3056113e17056125c89ae220bef7a/Templates/DefaultProject/Template/export.bat) script (or running the [`export.sh`](https://github.com/o3de/o3de/blob/480ce34ab7f3056113e17056125c89ae220bef7a/Templates/DefaultProject/Template/export.sh) script on Linux) in the root of your project folder.
```

Reply:
Not sure if we want to link to github the source for the scripts. Not really necessary, and not the exact same file considering these are templates and will have at least the engine root replaced when writing to the new project location.

Reply:
I also don't think you need to say 'assuming you are using the default level`, because the export script does not specify the level at all. They can still use their own level name, provided they update the loadlevel value in their load_level.setreg file that will be created from the template.

Reply:
i think i agree with your take here spham, b/c the script may morph over time.